src/stan/lang/grammars/semantic_actions_def.cpp

```diff
@@ -3252,9 +3314,16 @@ namespace stan {
     boost::phoenix::function<reset_var_scope> reset_var_scope_f;
 
     void trace::operator()(const std::string& msg) const {
+      //      std::cout << msg << std::endl;
```

Comment:
Add a comment about why this is commented out and indicating that it should be turned back on for debugging.  Eventually, this could be synched with the `DEBUG` flag that we're introducing.

Reply:
commented.